assets/app/scripts/services/data.js

```diff
@@ -42,7 +42,7 @@ angular.module('openshiftConsole')
     for (var i = 0; i < subAttrs.length; i++) {
       attrValue = attrValue[subAttrs[i]];
       if (!attrValue) {
-        return;
+        attrValue = "";
```

Comment:
1. I would check if (attrValue === undefined) to make sure we don't drop falsy values
2. I would not add objects to the map that are completely missing the indexed attribute (previous behavior)
3. It looks like we're only using secondaryAttr in podsByServiceByLabel... if we don't need it, let's drop it